android/src/main/java/com/reactnativebledidcomm/BleDidcommModule.kt

```diff
@@ -54,6 +54,7 @@ class BleDidcommModule(private val context: ReactApplicationContext) :
 
     @ReactMethod
     fun shutdownCentral(
+        @Suppress("UNUSED_PARAMETER") options: ReadableMap,
```

Comment:
Central needs no changes?

Reply:
Do you mean the same changes as in the `peripheralManager`? The central shutdown works fine without those changes.

Reply:
Not the same changes per se. As long as it is tested with the flow I mentioned above it is fine.